cmd/kogito/command/shared/install_operator.go

```diff
@@ -66,7 +65,7 @@ func MustInstallOperatorIfNotExists(namespace string, operatorImage string, clie
 		operatorImage = DefaultOperatorImageNameTag
 	}
 
-	if exists, err := checkKogitoOperatorExists(client, namespace); err != nil {
+	if exists, err := infra.CheckKogitoOperatorExists(client, namespace); err != nil {
```

Comment:
`infra` in French is a short cut for infrastructure? It's in Portuguese as well, but IDK if it's in English also. I'm just curious because I'm always tempting to use `infra` all the time instead. :p